README.md

```diff
@@ -12,6 +13,10 @@ To **run and develop** do the following:
 * *Optional:* run `tsc -w`, make code changes (on Windows, try `start node ".\node_modules\typescript\bin\tsc -w"`)
 * Press F5 to debug
 
+### Debugging
```

Comment:
1. This should go above 'Development'
2. As I think about this more, I wonder if we want to actually pull most of the content into this file instead - a lot of the content is really about installation which is common to any usage of the extension